cirq/google/engine/engine_client.py

```diff
@@ -766,7 +766,7 @@ def update_reservation(self,
         if end:
             reservation.end_time.seconds = int(end.timestamp())
             paths.append('end_time')
-        if whitelisted_users:
+        if whitelisted_users != None:
```

Comment:
Should we add a short comment in the doc string to denote the difference between [] and None?

Reply:
Done.